docs/installing.rst

```diff
@@ -55,8 +68,69 @@ Install behind a proxy::
   python get-pip.py --proxy="[user:passwd@]proxy.server:port"
 
 
-Upgrade pip
------------
+Installing with Linux Package Managers
+--------------------------------------
+
+Fedora
+~~~~~~
+
+To install the supported version of pip:
+
+* < Fedora 23:
+
+ * Python 2: ``sudo yum install python-pip``
+ * Python 3: ``sudo yum install python3-pip``
+
+* >= Fedora 23:
+
+ * Python 2: ``sudo dnf install python-pip``
+ * Python 3: ``sudo dnf install python3-pip``
+
+To get newer versions of pip (and also setuptools and wheel), you can enable the
+"unofficial" `PyPA Copr Repo <https://copr.fedoraproject.org/coprs/pypa/pypa/>`_
+using `these instructions
+<https://fedorahosted.org/copr/wiki/HowToEnableRepo>`__, and run the same
```

Comment:
Learned something, thanks :) and no strong opinion on the text link
(and sorry for the noise)